congress/base.py

```diff
@@ -7,6 +7,19 @@
 BASE_URL = 'https://api.propublica.org/congress/{version}'
 
 class Client:
+    """A client for making requests to the API
+
+    Parameters
+    ----------
+    api_key : str
+        An API key for authentication
+    parser : parser object
+        An object which implements a `parse` method that is capable of
+        handling a json object
+    validator : validator object
```

Comment:
for the parser object you explain what this object does while for the validator you don't.  I would prefer keeping the information about the parser interface in the Parser object.